File: src/main/java/it/frob/sleighidea/SleighLexerAdapter.java
// SPDX-License-Identifier: Apache-2.0

package it.frob.sleighidea;

import com.intellij.lexer.FlexAdapter;

public class SleighLexerAdapter extends FlexAdapter {
    public SleighLexerAdapter() {
        super(new SleighLexer(null));
    }
}

File: src/main/kotlin/it/frob/sleighidea/SleighLexerAdapter.kt
// SPDX-License-Identifier: Apache-2.0

package it.frob.sleighidea

import com.intellij.lexer.FlexAdapter
import it.frob.sleighidea.parser._SleighLexer

class SleighLexerAdapter : FlexAdapter(_SleighLexer(null))